### DISCUSSION TITLE
FYSETC Shaper mount: /mnt: special device /dev/sda1 does not exist

### DISCUSSION THREAD

**Discussion by AbitNutzT00** [Guilouz/Klipper-Flsun-Speeder-Pad#166] [General]
I'm wondering if the FYSETC shaper is bad. I plug it in and it will light up blue and the red light be on. Then all the lights go off. I can't get it to show up. I've tried 3 different cables. To be clear. I plug it in while holding down the button. wait for something to happen. Let the button go. Unplug it. Run the commands....and I get the "mount: /mnt: special device /dev/sda1 does not exist". I checked the menu options a million times. There are only 4 lines to it. Any insight would be helpful.

**Comment by AbitNutzT00** [comment]
It's fixed. It still will not find the shaper no matter what using the method in the tutorial. However, if you plug the USB cable into your Windows box it pops up right away and the file is in the out folder of the klipper directory.

**Comment by Hal9999** [comment]
You wrote: "Let the button go. Unplug it. Run the commands..."
You shouldn't unplug before launching the mount command
